pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -378,4 +407,23 @@ public ServerType getServerType() {
     @Deprecated
     public static final String TABLE_NAME = "segment.table.name";
   }
+
+  public enum UpdateSemantic {
+    APPEND,
```

Comment:
Please add comments on each of these. What does APPEND mean, vs what UPSERT means. Feel free to point to the design doc